### PR TITLE
Code Quality: Formatted code for readability 

### PR DESCRIPTION
**Pull request by cinqmilleans**
**Resolved / Related Issues**
Small pr that improves the readability of some small classes.

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [ ] Tested the changes for accessibility

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

**Review by QuaintMako — DISMISSED** [review]
LGTM.

**Review by ferrariofilippo — DISMISSED** [review]
LGTM

File: src/Files.App/Helpers/ResourceHelpers.cs [hunk 1]
```diff
@@ -6,16 +6,10 @@ namespace Files.App.Helpers
 	[MarkupExtensionReturnType(ReturnType = typeof(string))]
 	public sealed class ResourceString : MarkupExtension
 	{
-		private static ResourceLoader resourceLoader = new ResourceLoader();
+		private static readonly ResourceLoader loader = new();
```
Comment (yaira2) [inline src/Files.App/Helpers/ResourceHelpers.cs:9]: Readonly is alright but lets keep the names as they were before.

File: src/Files.App/Shell/ContextMenuItem.cs [hunk 2]
```diff
@@ -19,17 +16,12 @@ public void Dispose()
 
 		protected virtual void Dispose(bool disposing)
 		{
-			if (disposing)
+			if (disposing && SubItems is not null)
```
Comment (yaira2) [inline src/Files.App/Shell/ContextMenuItem.cs:19]: Why was the null protection removed?

File: src/Files.App/UserControls/Settings/SettingsBlockControl.xaml [hunk 3]
```diff
@@ -8,6 +8,7 @@
 	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 	d:DesignHeight="300"
 	d:DesignWidth="400"
+	Loaded="SettingsBlockControl_Loaded"
```
Comment (yaira2) [inline src/Files.App/UserControls/Settings/SettingsBlockControl.xaml:11]: The plan is to switch to the Toolkits control when it becomes available so let's leave the current control as is and revert these changes.

File: src/Files.App/Shell/ContextMenuItem.cs (hunk 2 above)
Comment (cinqmilleans) [reply src/Files.App/Shell/ContextMenuItem.cs:19]: The behavior has not changed. I just used a single if instead of 2 nested ones.

File: src/Files.App/UserControls/Settings/SettingsBlockControl.xaml (hunk 3 above)
Comment (cinqmilleans) [reply src/Files.App/UserControls/Settings/SettingsBlockControl.xaml]: It's done.

File: src/Files.App/Helpers/ResourceHelpers.cs (hunk 1 above)
Comment (cinqmilleans) [reply src/Files.App/Helpers/ResourceHelpers.cs]: It's done.

File: src/Files.App/UserControls/Settings/SettingsDisplayControl.xaml.cs [hunk 4]
```diff
@@ -7,69 +7,53 @@ namespace Files.App.UserControls.Settings
 	[ContentProperty(Name = nameof(SettingsActionableElement))]
 	public sealed partial class SettingsDisplayControl : UserControl
 	{
-		public FrameworkElement SettingsActionableElement { get; set; }
```
Comment (yaira2) [inline src/Files.App/UserControls/Settings/SettingsDisplayControl.xaml.cs:10]: Can you revert the changes to this file?

Comment (cinqmilleans) [reply src/Files.App/UserControls/Settings/SettingsDisplayControl.xaml.cs]: It's done.

**Review by ferrariofilippo — APPROVED** [review]
LGTM

**Review by d2dyno1 — APPROVED** [review]
